contracts/Vault.sol

```diff
@@ -311,16 +311,18 @@ contract Vault is
         _withdraw(_to, _ids, true);
     }
 
-    function investableAmount() public view returns (uint256) {
-        uint256 maxInvestableAssets = totalUnderlying().pctOf(investPct);
-
-        uint256 alreadyInvested = strategy.investedAssets();
-
-        if (alreadyInvested >= maxInvestableAssets) {
-            return 0;
+    function investState()
+        public
```

Comment:
Do we maybe want to make this external so the backend can quickly check before calling the updateInvested ?

Reply:
weird actually I just tried switching the public to external and I get an error

![image](https://user-images.githubusercontent.com/4612986/165288320-81e89fc7-6e83-4622-a4f7-092957b6043f.png)

we should be able to call this from the backend regardless though, public makes it available to call off chain

Reply:
I don't understand. being public makes it external as well. and if it's used internally (according to that screenshot), then `public` is the right one

Reply:
sorry, I was confused, you guys are right